src/runtime_config/constitution.js

```diff
@@ -104,6 +104,15 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "valid_pem",
+    new Action(
+      function (args) {
+        return ccf.isValidPem(args.pem);
```

Comment:
What is this checking exactly? Any PEM-style thing? Or just X509? Public key/private key?

Reply:
Anything that makes mbedtls_x509_crt_parse happy, so any chain of x509 certs. `isValidX509Chain`?